lock-googleplus/src/main/java/com/auth0/android/google/GoogleAuthHandler.java

```diff
@@ -0,0 +1,30 @@
+package com.auth0.android.google;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.auth0.android.authentication.AuthenticationAPIClient;
+import com.auth0.android.provider.AuthHandler;
+import com.auth0.android.provider.AuthProvider;
+
+public class GoogleAuthHandler implements AuthHandler {
+
+    private final GoogleAuthProvider provider;
+
+    public GoogleAuthHandler(@NonNull AuthenticationAPIClient apiClient, @NonNull String serverClientId) {
```

Comment:
We can remove this constructor to decouple a bit this class from the api client, and leave the one that receives the provider.